wesad/transformer.py

```
@@ -0,0 +1,320 @@
+    lines: list[str] = []
+    readme_path = subject_dir / f"{subject_id}_readme.txt"
+    if readme_path.exists():
+        lines.extend(line.strip() for line in readme_path.read_text(errors="replace").splitlines() if line.strip())
+    return lines
+
+
+def build_wesad_timef_row(subject_dir: str | Path) -> dict:
+    subject_dir = Path(subject_dir)
+    subject_id = subject_dir.name
+    with (subject_dir / f"{subject_id}.pkl").open("rb") as handle:
+        payload = pickle.load(handle, encoding="latin1")
```

Ruff (0.15.12)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)